docs/website/docs/guides/ml-frameworks/pytorch.md

```diff
@@ -5,197 +5,248 @@ tags:
   - Python
   - PyTorch
 icon: simple/pytorch
+status: new
 ---
 
-# PyTorch integration
+# PyTorch + IREE = :octicons-heart-16:
```

Comment:
maybe it's my firefox, but I see a question mark instead of a heart.

Reply:
I had a question mark there before. Just pushed a new version with the icon (should show up in a few minutes on the preview site)